xc/device.ml

```diff
@@ -942,13 +1009,13 @@ module Vgpu = DaemonMgmt(struct let pid_path domid = sprintf "/local/domain/%d/v
 module PCI = struct
 
   type t = {
-    address: Xenops_interface.Pci.address;
+    address: Xcp_pci.address;
```

Comment:
I don't think these renames were needed (anymore)? We took care of that in the IDL.

Reply:
Fixed all of them. Thanks!
And I squashed 5 commits into one since I think there are too many of them to review it clearly.